packages/cli/src/info/cdk_info.ts

```diff
@@ -0,0 +1,62 @@
+import { execa as _execa } from 'execa';
+import { indent } from './indent.js';
+
+/**
+ * Format CDK information.
+ * @param execa - The execa to run the cdk command.
+ * @returns The cdk doctor output.
+ */
+export const getCdkInfo = async (execa = _execa): Promise<string> => {
+  const cdkDoctorArgs: string[] = ['cdk', 'doctor', '--', ' --no-color'];
+  let output;
+  try {
+    // type-cast the value to string, we know it will be a string given the options
+    output = (
+      await execa('npx', cdkDoctorArgs, {
+        stderr: 'pipe',
+      })
+    ).stderr;
+  } catch (error) {
+    // should this be replaced with an AmplifyError
+    throw new Error(`Failed to run command: ${error as string}`);
+  }
```

Comment:
Please remove try catch block. This error handling doesn't bring any value.
If cdk doctor doesn't work this would be a fault not AmplifyError anyway.